src/auro-sidenav.js

```diff
@@ -138,12 +144,52 @@ export class AuroSidenav extends LitElement {
     this.addEventListener('mousedown', this.handleMouseDown);
   }
 
+  async updated(changedProperties) {
```

Comment:
```suggestion
  /**
 * Asynchronously handles updates to the component.
 * This method is automatically called when properties have been updated.
 * @param {Map<string | number | symbol, unknown>} changedProperties - A Map of properties that have changed, where the key is the property name and the value is its previous value.
 * @returns {Promise<void>} A Promise that resolves when the update is complete.
 * @override
 */
  
  async updated(changedProperties) {
```

I am unsure if this is private or part of the public API?

Reply:
The method `updated` is a built-in Lit feature that runs during a particular lifecycle that seemed appropriate for our use-case.